casa/IO/MultiFile.h

```diff
@@ -130,17 +194,64 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     // Fsync the file (i.e., force the data to be physically written).
     virtual void fsync();
 
+    // Show some info.
+    void show (std::ostream&) const;
+
+    // Compress a block index by looking for subsequent block numbers.
+    static vector<Int64> packIndex (const vector<Int64>& blockNrs);
+
+    // Decompress a block index by inserting subsequent block numbers.
+    static vector<Int64> unpackIndex (const vector<Int64>& blockNrs);
+
   private:
+    // Copy constructor and assignment not possible.
+    MultiFile (const MultiFile&);
+    MultiFile& operator= (const MultiFile&);
+
+    // Initialize the MultiFile object.
+    void init (ByteIO::OpenOption option);
+    // Read the file info for the new version 2.
+    void getInfoVersion2 (Int64 contBlockNr, CanonicalIO& aio);
+    // Store the CRC of a data block in the index.
+    void storeCRC (const void* buffer, Int64 blknr);
+    // Check the CRC of a data block read.
+    void checkCRC (const void* buffer, Int64 blknr) const;
+    // Calculate the CRC of a data block.
+    uInt calcCRC (const void* buffer, Int64 size) const;
+    // Fill the precalculated CRC byte table.
+    void fillCRCTable();
+    // Write a vector of Int64.
+    void writeVector (CanonicalIO& cio, const vector<Int64>& index);
```

Comment:
I do not understand what you mean with unqualified. They are private member functions.

Reply:
unqualified `vector` instead of `std::vector`. I really dislike this style since in my experience it can give a lot of issues when switching to a new C++ version. Suddenly the compiler may get errors due to ambiguous code. This happens mostly with `using namespace std`, which seems to be only done in `.cc` files. But `using std::foo` may give issues when a new overload is added to the `std` namespace.

Reply:
changed to std::vector